### PR TITLE
GAWB-1688:  Make file previews disappear for empty files

### PR DESCRIPTION
**Pull request by ansingh7115**
What the file previews looked like before when the file was empty:

screen shot 2017-03-02 at 4 33 45 pm

What the file previews will look like now when the files are empty: 

screen shot 2017-03-02 at 2 49 25 pm

- [x] **Submitter**: Include the JIRA issue number in the PR description
- [x] **Submitter**: Check documentation and code comments. Add explanatory PR comments if helpful.
- [ ] **Submitter**: JIRA ticket checks:
  * Acceptance criteria exists and is met
  * Note any changes to implementation from the description
  * Add notes on what you've tested
- [x] **Submitter**: Update RC_XXX release ticket with any config or environment changes necessary
- [x] **Submitter**: Update FISMA documentation if changes to:
  * Authentication
  * Authorization
  * Encryption
  * Audit trails
- [x] **Submitter**: If you're adding new libraries, sign us up to security updates for them
- [x] Tell the tech lead (TL) that the PR exists if they wants to look at it
- [x] Anoint a lead reviewer (LR). **Assign PR to LR**
* Review cycle:
  * LR reviews
  * Rest of team may comment on PR at will
  * **LR assigns to submitter** for feedback fixes
  * Submitter rebases to develop again if necessary
  * Submitter makes further commits. DO NOT SQUASH
  * Submitter updates documentation as needed
  * Submitter **reassigns to LR** for further feedback
- [ ] **TL** sign off
- [ ] **LR** sign off
- [ ] **Product Owner** sign off
- [ ] **Assign to submitter** to finalize
- [ ] **Submitter**: Verify all tests go green, including CI tests
- [ ] **Submitter**: Squash commits and merge to develop
- [ ] **Submitter**: Delete branch after merge
- [ ] **Submitter**: **Test this change works on dev environment after deployment**. YOU own getting it fixed if dev isn't working for ANY reason!
- [ ] **Submitter**: Mark JIRA issue as resolved once this checklist is completed

### CODE REVIEW

File: src/cljs/main/broadfcui/common/gcs_file_preview.cljs [hunk 1]
```diff
@@ -28,34 +28,40 @@
           (labeled "Google Bucket" (:bucket-name props))
           (labeled "Object" (:object props))
           [:div {:style {:marginTop "1em"}}
-           [:div {} "Previews for some filetypes may be unsupported. "]
+           [:div {} (when-not (or (= data-size "0") (clojure.string/blank? data-size))
```
Comment (MattPutnam) [inline src/cljs/main/broadfcui/common/gcs_file_preview.cljs]: We're checking `(or (= data-size "0") (clojure.string/blank? data-size))` several times, so let's pull this test out into a common function.

**Review by MattPutnam — APPROVED** [review]
👍 with one refactoring

**Comment by dmohs** [comment]
Thanks for the screenshots! +1 to what Putnam said. Also, it's weird that you have to check for both "0" and blank, so a comment about why that is would be helpful (if you know).

**Comment by ansingh7115** [comment]
I tried implementing it with only checking for "0" and not blank - and the behavior I saw was that the preview stuff would load and then a split-second later would disappear. The reason for this was that the data-size was nil for that split-second and then would be populated by the actual size. 

I decided to use blank? because it handles nil, empty, or only whitespace cases.

**Pull request merged**